docs/style/class.pu

```diff
@@ -0,0 +1,20 @@
+@startuml stylesheet
+
```

Comment:
questo stile va incluso in tutti gli altri diagrammi (mi pare il comando sia `!include ./relative/path/to/style`)